solidity/contracts/GovernanceUtils.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
```

Comment:
We test `GovernanceUtils` indirectly on the occasion of testing `VendingMachine`. We should have separate unit tests specific to cover `GovernanceUtils` functions.

Reply:
I would rather leave it as-is and think about testing `GovernanceUtils` when we add another implementation using it.

Reply:
> I would rather leave it as-is and think about testing GovernanceUtils when we add another implementation using it.
 
Up to you! I'm 99% sure we won't do it later 😉